input.go

```diff
@@ -41,9 +97,48 @@ func (t *Terminal) TypedKey(e *fyne.KeyEvent) {
 		_, _ = t.in.Write([]byte{asciiEscape, cursorPrefix, 'D'})
 	case fyne.KeyRight:
 		_, _ = t.in.Write([]byte{asciiEscape, cursorPrefix, 'C'})
+	case fyne.KeyPageUp:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', '5', '~'})
+	case fyne.KeyPageDown:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', '6', '~'})
+	case fyne.KeyHome:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', '1', '~'})
+	case fyne.KeyInsert:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', '2', '~'})
+	case fyne.KeyEnd:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', '4', '~'})
+	}
+
+}
+
+func (t *Terminal) TrackKeyboardState(down bool, e *fyne.KeyEvent) {
+
+	// TODO these were discovered on the mac though debugging this method
+	//   double check with windows
+	switch e.Physical.ScanCode {
```

Comment:
Why not use `e.Name` again? It would remove the somewhat OS dependent scan codes.

Reply:
The names come though as "LeftShift" "RightShift" etc. 
This will require changes in the Fyne package. I will PR these changes and comment back here